src/navigation/HomeNavigation.js

```diff
@@ -20,27 +27,40 @@ const TAB_SCREEN_OPTIONS = {
 	lazy: false //added to ensure that player loads correctly on all tabs
 };
 
-const HomeNavigation = () => (
-	<Tab.Navigator screenOptions={TAB_SCREEN_OPTIONS}>
-		{HOME_SCREENS_ARRAY.map((item, index) => (
-			<Tab.Screen
-				key={index}
-				name={item.name}
-				tabIcon="home"
-				options={{
-					tabBarIcon: ({ color, size }) => (
-						<MaterialCommunityIcons
-							name={item.tabIcon}
-							color={color}
-							size={size}
-						/>
-					)
-				}}
-			>
-				{item.component}
-			</Tab.Screen>
-		))}
-	</Tab.Navigator>
-);
+const HomeNavigation = () => {
+	const user = useSelector(({ user }) => user?.user?.user);
```

Comment:
this shouldn't require a user?.user?.user. 

you should rename the slice when it goes into the store into UserSlice, then it should have a user variable in there, so this should just be 


const user = userSelector(({UserSlice}) => UserSlice?.user)